scripts/deploy-mock.js

```diff
@@ -150,7 +150,7 @@ async function main() {
   /// =======  xCitadelLocker
   await xCitadelLocker
     .connect(governance)
-    .initialize(address(xCitadel), "Vote Locked xCitadel", "vlCTDL");
+    .initialize(address(xCitadel), address(gac), "Vote Locked xCitadel", "vlCTDL");
```

Comment:
this needs to be added in deploy-local as well unless its already fixed there

Reply:
It was making some other problems there, I will do complete cleanup of scripts later today